src/anyvar/translate/variation_normalizer.py

```diff
@@ -25,21 +30,68 @@ def __init__(self, endpoint_uri: str):
         """Emit normalization request. Broken out to enable mocking.
 
         :param request_url: URL containing normalization request parameters
+        :return: content of response
         """
         return requests.get(request_url)
 
-    def translate(self, var: str, **kwargs: Dict) -> vrs_models.Allele:
+    def get_sequence_id(self, accession_id: str) -> str:
+        """Get GA4GH sequence identifier for provided accession ID
+
+        :param accession_id: ID to convert
+        :return: equivalent GA4GH sequence ID
+        :raise: KeyError if no equivalent ID is available
+        """
+        req_url = self.endpoint_base + f"translate_identifier?identifier={accession_id}&target_namespaces=ga4gh"  # noqa: E501
+        resp = self._send_rest_request(req_url)
+        if resp.status_code == 404:
+            raise TranslatorConnectionException(
+                f"Failed to get response from Variation Normalizer REST endpoint at {req_url}")  # noqa: E501
+        elif resp.status_code == 500:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned server error for {accession_id}"  # noqa: E501
+            )
+        elif resp.status_code != 200:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned {resp.status_code} for {accession_id}"  # noqa: E501
+            )
+
+        resp_json = resp.json()
+        if resp_json.get("warnings") or len(resp_json.get("aliases", [])) == 0:
+            raise KeyError(f"Unable to find GA4GH ID for {accession_id}")
+        return resp_json["aliases"][0]
+
+    @staticmethod
+    def _safe_check_variation_type(var_normalizer_response: Dict) -> Optional[str]:
+        """Safely check variation type in response from variation normalizer. Resilient
+        to different response structures.
+
+        :param var_normalizer_response: complete response received from normalizer
+        :return: type value if exists, None otherwise
+        """
+        variation_descriptor = var_normalizer_response.get("variation_descriptor")
+        if not variation_descriptor:
+            return None
+        variation = variation_descriptor.get("variation")
+        if not variation:
+            return None
+        return variation.get("type")
+
+    def translate(
+        self, var: str, untranslatable_to_text: bool, **kwargs: Dict
+    ) -> vrs_models.Allele:
         """Translate provided variation text into a normalized VRS object.
 
         :param var: user-provided text object describing or referencing a variation.
+        :param untranslatable_to_text: if True, store un-normalized descriptions
+            as Text objects
         :raises TranslatorConnectionException: if translation request returns error
         """
 
-        req_url = self.endpoint_base + f"normalize?q={var}"
+        req_url = self.endpoint_base + f"normalize?q={var}&untranslatable_returns_text=true"  # noqa: E501
```

Comment:
Did we want to hard code this? 
```suggestion
        req_url = self.endpoint_base + f"normalize?q={var}&untranslatable_returns_text={untranslatable_to_text}"  # noqa: E501
```